### ISSUE TITLE
Consider adopting tool (howfairis) to generate NIST badge

### ISSUE THREAD

**Issue by wd15** [usnistgov/opensource-repo#6]
https://github.com/fair-software/howfairis

This is an example of a tool that examines available badges and then generates a meta badge describing if subcategories have been satisfied. Might be worth considering for NIST if relatively straightforward to adopt for our categories of interest that are FAIR-like, but might have other requirements.

**Comment by tkphd** [comment]
Badges are fine... I don't see all too many of them.
I prefer the [F-UJI analyzer](https://www.f-uji.net), which generates a nice report with a score. Currently, [this repo is at 14% compliant](https://www.f-uji.net/view/67), with a checklist of specific actions we can take. Noit as shiny as a badge, but more useful, I think.

**Comment by wd15** [comment]
> I prefer the [F-UJI analyzer](https://www.f-uji.net), which generates a nice report with a score. Currently, [this repo is at 14% compliant](https://www.f-uji.net/view/67), with a checklist of specific actions we can take. Noit as shiny as a badge, but more useful, I think.

That's really nice. Note that the F-UJI thing isn't specifically for software (unless I'm mistaken). Right, it doesn't have to be a badge, but any automated mechanism. For example, opening an issue or a PR, whatever to notify developers that their repo is missing something.

**Comment by tkphd** [comment]
Indeed, F-UJI is a generic FAIR test: software, data, kitten pics, doesn't matter.

Repos don't *have* to be FAIR until they're "published," at which point the Data Sponsor is responsible for checking. However, yeah, it would be cool to configure the default test suite to run a FAIR-proofer on the repo so there's an indication. Then FAIR is opt-out to get test passing, rather than opt-in and therefore ignored.
